app/hotels/src/singleHotel/roomList/RoomBadges.js

```diff
@@ -60,7 +57,32 @@ export const RoomBadges = (props: Props) => {
     );
   }
 
-  return badges;
```

Comment:
I think you forgot to clean this file. It looks like `badges` is defined for no reason now and the code is almost duplicated.

Reply:
good catch 😄